app/src/main/java/org/mozilla/fenix/share/ShareViewModel.kt

```diff
@@ -87,15 +104,44 @@ class ShareViewModel(application: Application) : AndroidViewModel(application) {
         }
     }
 
+    private fun filterRecentApps(
```

Comment:
Maybe a clearer name would be `filterOutRecentApps`?

Reply:
The 'Out' actually makes it much clearer, thanks! Used a filter here also instead of the for loop